src/sentry/replays/usecases/issue.py

```diff
@@ -0,0 +1,53 @@
+import datetime
+import uuid
+from typing import Any, Dict, Type
+
+from sentry.issues.grouptype import GroupType
+from sentry.issues.issue_occurrence import IssueOccurrence
+from sentry.issues.producer import produce_occurrence_to_kafka
+
+
+def new_issue_occurrence(
+    environment: str,
+    fingerprint: str,
+    issue_type: Type[GroupType],
+    platform: str,
+    project_id: int,
+    release: str,
+    subtitle: str,
+    timestamp: datetime.datetime,
+    title: str,
+    extra_event_data: Dict[str, Any],
+) -> None:
+    """Produce a new issue occurence to Kafka."""
+    event_id = uuid.uuid4().hex
+    extra_event_data["event_id"] = event_id
+
+    occurrence = IssueOccurrence(
+        id=uuid.uuid4().hex,
```

Comment:
what's the difference between `id` and `event_id` -- should either of them have some kind of deterministic value based on the event, or is random ok?

Reply:
I'm pretty sure random is okay.  `event_id` and `event["event_id"]` should match but otherwise I think `id` can be whatever.

@wedamija Is this correct?